Ship_Game/GameScreens/ShipDesign/ShipDesignScreen.cs

```diff
@@ -413,6 +413,8 @@ public override void Update(float fixedDeltaTime)
             DesignedShip.Velocity = new Vector2(0, 100);
             DesignedShip.UpdateThrusters(simTime);
 
+            ScreenManager.StartMusic("ShipyardTheme");

```

Comment:
Why do we need to Start Music every update?
I see we are starting the shipyard music in the LoadContent.

Reply:
because it is responsible if music has ended and should restart

Reply:
StartMusic is just copy of previous logic, so nothing exceptionally new

Reply:
actually you might be right

Reply:
I dont see any part in the code that restarts the music after the playback us finished

Reply:
Ok, so it is needed in update in order to restart the music when it ends.  
Just need to fix the bug with the restart not working (see above suggestion)